privacyidea/lib/policy.py

```diff
@@ -161,9 +161,9 @@
 import logging
 from ..models import (Policy, Config, PRIVACYIDEA_TIMESTAMP, db,
                       save_config_timestamp)
-from flask import current_app
 from privacyidea.lib.config import (get_token_classes, get_token_types,
                                     Singleton)
+from privacyidea.lib.framework import get_app_config_value
```

Comment:
Is there a reason why You sometimes use `from .framework import ...` and sometime `from privacyidea.lib.framework import ...`?

Reply:
No, I think some of these are from my IDE and some are handwritten :) What style would be better?

Reply:
Personally I prefer absolute paths/modules to relative ones. I don't know if there is a *pythonic* way. At least we should be consistent I think.

Reply:
A lot of style guides seem to recommend absolute imports too ([example](https://realpython.com/absolute-vs-relative-python-imports/#pros-and-cons-of-absolute-imports)). So I changed all new imports to absolute imports in this commit. However, this isn't consistent with other imports in the respective files :-) But I guess that's ok for now?